static/src/javascripts-legacy/projects/common/modules/experiments/tests/bundles-landing-page.js

```diff
@@ -0,0 +1,91 @@
+define([
+    'bean',
+    'fastdom',
+    'qwery',
+    'lib/config',
+    'lib/detect',
+    'lib/mediator',
+    'lib/$'
+
+], function (
+    bean,
+    fastdom,
+    qwery,
+    config,
+    detect,
+    mediator,
+    $
+) {
+    return function () {
+        var self = this;
+
+        this.id = 'BundlesLandingPage';
+        this.start = '2017-03-24';
+        this.expiry = '2017-04-06'; // Thursday 6th April
+        this.author = 'Justin Pinner';
+        this.description = 'Route 10% of epic traffic to recurring contributions-enabled bundle page';
+        this.showForSensitive = true;
+        this.audience = 0.1;  // 10% (of epic-seeing readers)
+        this.audienceOffset = 0;
+        this.successMeasure = 'Two thousand conversions of any kind';
+        this.audienceCriteria = 'Epic click-throughs';
+        this.dataLinkNames = '';
+        this.idealOutcome = 'We score some recurring contributions and don\'t miss out on others';
+        this.hypothesis = 'People want recurring contributions, and the landing page serves all options';
+
+        this.canRun = function () {
+            return config.page.edition.toUpperCase() === 'UK';
```

Comment:
We don't check edition anymore? We only want to target UK edition readers, rather than anyone who happens to be in the UK _I think_. I'll double-check.